src/pages/academy/grading/subcomponents/GradingSubmissionsTable.tsx

```diff
@@ -410,6 +414,25 @@ const GradingSubmissionTable: React.FC<GradingSubmissionTableProps> = ({
           suppressPaginationPanel={tableProperties.suppressPaginationPanel}
           suppressRowClickSelection={tableProperties.suppressRowClickSelection}
           domLayout="autoHeight"
+          onFilterChanged={e => {
+            if (!e.afterFloatingFilter) {
+              return;
+            }
+            const filters = e.api.getFilterModel();
+            const cellFilters = [];
+            for (const [key, { filter: query }] of Object.entries(filters)) {
+              switch (key) {
+                // Fields that BE supports filtering on
+                case ColumnFields.studentName:
+                case ColumnFields.studentUsername:
+                case ColumnFields.groupName:
+                  console.log(query);
```

Comment:
Do we need this here?

Reply:
Oops

Reply:
Resolved in 59421b029fe484b126e584196b3e4b39322a0e7a.